analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -198,22 +183,45 @@ def __init__(self, disable_processing, io_file_percent):
         self.io_file_percent = io_file_percent
 
     def only_do_IO(self, events):
-        # standard AGC branches cover 4% of the data
-            branches_to_read = ["jet_pt", "jet_eta", "jet_phi", "jet_btag", "jet_e", "muon_pt", "electron_pt"]
-            if self.io_file_percent not in [4, 15, 25, 50]:
-                raise NotImplementedError("supported values for I/O percentage are 4, 15, 25, 50")
-            if self.io_file_percent >= 15:
-                branches_to_read += ["trigobj_e"]
-            if self.io_file_percent >= 25:
-                branches_to_read += ["trigobj_pt"]
-            if self.io_file_percent >= 50:
-                branches_to_read += ["trigobj_eta", "trigobj_phi", "jet_px", "jet_py", "jet_pz", "jet_ch"]
-
+        # standard AGC branches cover 2.7% of the data
```

Comment:
It's somewhat counterintuitive to me that we read a smaller fraction of data using the same branches, even though the file size also decreased. I imagine it must be that the number of jets / leptons per events are smaller in the NanoAODs since the previous ntuples probably had less selection requirements applied to them already?

Reply:
Maybe. Weren't there also many duplicate counts branches in the old files? Maybe this is related.

Reply:
will investigate more closely in a follow-up issue or PR